systemd/systemd_test.go

```diff
@@ -604,20 +808,43 @@ NeedDaemonReload=no
 }
 
 func (s *SystemdTestSuite) TestStopTimeout(c *C) {
```

Comment:
is this test different from others now, should we keep it? it originally was about explicit timeouts, a feature we removed. If it's are different should it have a changed name?

Reply:
There are 3 possible returns from Stop()

1. nil: If all services stopped without any problems
2. "error from systemctl" if stop or show returned an error
3. "timeout" if no error, but services not showing as stopped

The two tests mentioned here tests the "timeout" error code path. This path means no error returned by systemctl commands, but still systemctl show reports units are not stopped.
.